### PR TITLE
feat: add alert component

### PR DESCRIPTION
**Pull request by Kavita539**
Added Alert Component.
It has two types of alert : Simple Alert and Outlined Alert
Can also preview the live layout on netlify deployed preview.

Layout of Alerts (Simple Alert):
image

Layout of Alerts (Outlined Alert):
image

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.alert{
+    width: 100%;
+    border-radius: 4px;
+    padding: 1rem 1.5rem;
+    margin: .5rem 0;
```
Comment (Prabhav-Sharma) [inline components/alert/alert.css:5]: Use global variables for properties like margin, padding and border-radius to ensure consistency across the website.

**Review by Prabhav-Sharma — CHANGES_REQUESTED** [review]
The contrast for the text and background in the success, warning and error alert is too low, might want to change text color to white or something which adds more contrast

File: components/alert/alert.css (hunk 1 above)
Comment (Kavita539) [reply components/alert/alert.css:5]: Have made the changes, plz do review!

**Review by Prabhav-Sharma — APPROVED** [review]

**Review by nikhilmalviya-dev — APPROVED** [review]
good work

**Review by msdeshmukh009 — APPROVED** [review]
Looks good to me.

**Pull request merged**